lib/action_cable.rb

```diff
@@ -18,8 +18,6 @@
 require 'action_cable/engine' if defined?(Rails)
 
 module ActionCable
-  VERSION = '0.0.3'
```

Comment:
We should require the new file, so the constant is still available to a running application.